README.md

```diff
@@ -30,10 +30,10 @@ Instantiate the population class with a death rate = 0.000006, and initial popul
 ``` python
 population = Population(death=0.000006, size=150000, birth_function=birth_rate_function)
 ```
-Generate two strains to model. Strain I1 has a death rate, alpha=0.00001, recovery rate, nu=0.143, and a transmission rate, beta=0.000003. This gives an R0 of 3.14. At the start of the simulation 150 individuals are infected with strain I1. Strain I2 has a death rate, alpha=0.002, recovery rate, nu=0.2, and a transmission rate, beta=0.00000429. This gives an R0 of 3.22. At the start of the simulation 10 individuals are infected with strain I2.
+Generate two strains to model. Strain I1 has a case fatality rate, CFR = 0.00007, recovery time = 7 days, and an R0 = 3.14. At the start of the simulation 150 individuals are infected with strain I1. Strain I2 has a case fatality rate, CFR = 0.001, recovery time = 8 days, and an R0 = 4.22. At the start of the simulation 10 individuals are infected with strain I2.
```

Comment:
Be consistent so do "case fatality rate (CRF) = 0.00007, recovery time = 7 days, and R0 = 3.14. Do we need to explain here what R0 means? Or do we do that somewhere else?